.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Issue_Template
+about: Create a useable template format for guidance
+title: '[ISSUES]'
+labels: 'Issue'
+assignees: ''
+
+---
+
+**Description of the issue**
+Describe the issue and share important details such as the desired solution, error messages, and where the issue originates.
+
```

Comment:
**Files that need to be updated** (something along those lines)
Describe what files or directories that would need to be changed / updated to fix issue.
(Im saying this should be different then the description.)

Reply:
Gotcha, I'll add this